pyworkflow/webservices/repository.py

```diff
@@ -41,9 +41,9 @@ def upload(self, jsonFileName):
 
         # we are going to upload a file so this is a multipart
         # connection
-        import poster.streaminghttp
-        datagen, headers = poster.encode.multipart_encode(params)
-        opener = poster.streaminghttp.register_openers()
+        import poster3.streaminghttp
+        datagen, headers = poster3.encode.multipart_encode(params)
```

Comment:
@pconesa params needs to be a list as per poster 3 documentation:
```
    ``params`` should be a sequence of (name, value) pairs or MultipartParam
    objects, or a mapping of names to values.
    Values are either strings parameter values, or file-like objects to use as
    the parameter value.  The file-like objects must support .read() and either
    .fileno() or both .seek() and .tell().
```

Reply:
hm, I am wrong, datagen, headers = multipart_encode( {"key": "value1"} ) should also work..

Reply:
I fixed this without poster.